pkg/virt-handler/hotplug-disk/mount.go

```diff
@@ -452,8 +454,8 @@ func (m *volumeMounter) volumeStatusReady(volumeName string, vmi *v1.VirtualMach
 	return true
 }
 
-func (m *volumeMounter) getSourceMajorMinor(sourceUID types.UID, volumeName string) (uint64, os.FileMode, error) {
-	basePath, err := deviceBasePath(sourceUID)
+func (m *volumeMounter) getSourceMajorMinor(sourceUID types.UID, volumeName string, kubeletPodsDir string) (uint64, os.FileMode, error) {
+	basePath, err := deviceBasePath(sourceUID, kubeletPodsDir)
```

Comment:
Basically, you do not need to pass `kubeletPodsDir` as an argument here as it is already available as a member of `m *volumeMounter` struct:
```suggestion
func (m *volumeMounter) getSourceMajorMinor(sourceUID types.UID, volumeName string) (uint64, os.FileMode, error) {
	basePath, err := deviceBasePath(sourceUID, m.kubeletPodsDir)
```
In such way you will have a more compact diff and will not need so many changes in the unit tests.